docs-web/src/test/java/com/sismics/docs/rest/TestReviewResource.java

```diff
@@ -0,0 +1,49 @@
+package com.sismics.docs.rest;
+
+import com.sismics.util.filter.TokenBasedSecurityFilter;
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.json.JsonObject;
+import javax.json.JsonValue;
+import javax.json.JsonWriter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Tests the ReviewResource.java file
+ *
+ * @author aruiz2
+ */
+
+public class TestReviewResource extends BaseJerseyTest {
+    private static final Logger log = LoggerFactory.getLogger(TestReviewResource.class);
```

Comment:
Since you are testing from the hardcoded files, try running mvn test from the main branch. If these tests are run before the ones I wrote, then that's fine. If they run after, it may cause problems because my tests also add things to the database for testing purposes.

Reply:
Its fine, they are all passing.

Reply:
Lol, that works too